src/azure-cli-core/azure/cli/core/parser.py

```diff
@@ -179,6 +182,14 @@ def enable_autocomplete(self):
         argcomplete.autocomplete(self, validator=lambda c, p: c.lower().startswith(p.lower()),
                                  default_completer=lambda _: ())
 
+    def _set_manual_recommendations(self, **kwargs):
+        recommendations = []
+        # recommendation for --query value error
+        if 'message' in kwargs and '--query' in kwargs['message']:
+            recommendations.append('To learn more about [--query JMESPATH] usage in AzureCLI, '
+                                   'visit https://aka.ms/CLIQuery')
```

Comment:
I can see from aka.ms that there are a few owners:

![image](https://user-images.githubusercontent.com/4003950/83382823-8670da00-a416-11ea-8553-b14e3775118d.png)

I am not sure whether using short link is a good practice as it is difficult to maintain. I am open to this.

Reply:
Lets just use the convention.

Reply:
Please investigate whether we can set link owner with a group name.